### PR TITLE
Add documentation issue template and button to edit documentation.

### PR DESCRIPTION
**Pull request by tonygardella**
## Description
There is a need to make documentation easier to fix by newcomers. I've added the button to edit and added an issue template to help make things easier. The button brings you to github whee you use the github interface to directly edit the .Rmd file and make a pull request.

@dlebauer  You're tagged with me on code review this week. @KristinaRiemer @bailsofhay You've recently thought of making changes to documentation so let me know if any more information should be added.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
@tonygardella so the language in the template is going to be the text in an issue when someone first opens one?

**Comment by tonygardella** [comment]
@KristinaRiemer @dlebauer Could I get a quick review on this so we can merge?

File: .github/ISSUE_TEMPLATE/Doc_changes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Documentation Changes
+about: Create a report to help us improve
+
+---
+
+### Document Change needed
+
+A clear description of what changes are needed with the location of the error that needs to be fixed
```
Comment (KristinaRiemer) [inline .github/ISSUE_TEMPLATE/Doc_changes.md]: What about something like "A clear description of what changes can be applied to the documentation in order to either make the documentation clearer and easier to follow, or to fix errors in the provided instructions or code."

File: .github/ISSUE_TEMPLATE/Doc_changes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+about: Create a report to help us improve
+
+---
+
+### Document Change needed
+
+A clear description of what changes are needed with the location of the error that needs to be fixed 
+
+Example:
+In the `master` version of the documentation there is an error in section "13.2.1", the offending line in 17 and can be found in
```
Comment (KristinaRiemer) [inline .github/ISSUE_TEMPLATE/Doc_changes.md]: Here's a slightly more specific example of a possible error. "Example: There is a broken URL link in the documentation. It is in the `master` version in section 13.2.1 on line 17. The underlying file is `/pecan/book_source/04_advanced_user_guide/01_R_workflow.Rmd`. The correct link should be https://aws.amazon.com/."

File: .github/ISSUE_TEMPLATE/Doc_changes.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+the file `/pecan/book_source/04_advanced_user_guide/01_R_workflow.Rmd' 
+
+### Information for those wanting to make changes
+
+Please follow the instructions in the [Editing this Book](https://pecanproject.github.io/pecan-documentation/develop/GettingStarted.html) section of the documentation.
```
Comment (KristinaRiemer) [inline .github/ISSUE_TEMPLATE/Doc_changes.md]: I think the website link is supposed to be [this](https://pecanproject.github.io/pecan-documentation/develop/bookediting.html)?

File: .github/ISSUE_TEMPLATE/Doc_changes.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Example:
+In the `master` version of the documentation there is an error in section "13.2.1", the offending line in 17 and can be found in
+the file `/pecan/book_source/04_advanced_user_guide/01_R_workflow.Rmd' 
+
+### Information for those wanting to make changes
+
+Please follow the instructions in the [Editing this Book](https://pecanproject.github.io/pecan-documentation/develop/GettingStarted.html) section of the documentation. 
+
+All R-markdown files related to documentation are located within the main directory `/pecan/book_source'
```
Comment (KristinaRiemer) [inline .github/ISSUE_TEMPLATE/Doc_changes.md]: I would replace this with "All R-markdown files related to documentation are located within the main directory `/pecan/book_source` except for the demos and vignettes which are in `pecan/documentation/tutorials`."

**Pull request merged**